.github/workflows/ci.yml

```diff
@@ -87,8 +87,13 @@ jobs:
       with:
         node-version: ${{ env.NODE_VERSION }}
 
-    - name: Run tests
-      run: EFFEKT_VALGRIND=1 EFFEKT_DEBUG=1 sbt clean test
+    - name: Run tests with coverage
+      run: EFFEKT_VALGRIND=1 EFFEKT_DEBUG=1 sbt clean coverage test coverageReport
+
+    - name: Upload test coverage to Codecov
+      uses: codecov/codecov-action@v5
+      with:
+        token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
As far as I know, this should _not_ fail builds ever, just upload the coverage report...